app/views/shared/_footer_lite.html.erb

```diff
@@ -47,14 +47,14 @@
       </div>
       <div class="display-flex flex-align-center flex-justify-center">
         <% if show_language_dropdown %>
-          <ul class='list-reset display-none tablet:display-block margin-bottom-0'>
+          <ul class='usa-list--unstyled display-none tablet:display-block margin-bottom-0'>
             <li class="i18n-desktop-toggle display-flex margin-y-1 margin-x-2">
```

Comment:
Two things:

1. `usa-list--unstyled` includes a `margin-bottom: 0` style, so the `margin-bottom-0` could be removed
2. Apparently `usa-list--unstyled` also unsets margin on its list item children, so this is causing a small visual regression for the language picker in the footer

One option could be to substitute margin on the `li` with padding on the `ul`:

```suggestion
          <ul class='usa-list--unstyled display-none tablet:display-block padding-y-1 padding-x-2'>
            <li class="i18n-desktop-toggle display-flex">
```